src/NServiceBus.AcceptanceTests/UnicastRoutingExtensions.cs

```diff
@@ -0,0 +1,22 @@
+namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Collections.Generic;
+    using Configuration.AdvanceExtensibility;
+    using NServiceBus.Routing;
+    using NServiceBus.Routing.MessageDrivenSubscriptions;
+
+    static class UnicastRoutingExtensions
+    {
+        public static void RegisterPublisher(this RoutingSettings config, Type eventType, string publisherEndpoint)
```

Comment:
I think we can remove this and use the `MessageDrivenPubSubRouting()` extension instead which let's you use the end user routing API. We should probably add an extension which returns a regular RoutingSettings too for non-pub/sub related test setups.